src/Transport/Addressing/Sanitization/ValidationResult.cs

```diff
@@ -1,29 +1,40 @@
 ﻿namespace NServiceBus.Transport.AzureServiceBus
 {
+    /// <summary>
+    /// Validation result to provide information about validation performed by the configured <see cref="ISanitizationStrategy"/>.
+    /// </summary>
     public class ValidationResult
     {
+        /// <summary></summary>
```

Comment:
All empty comments?

Reply:
This should never be exposed to begin with, but it is 🙁